testing/docker/bookinfo.yaml

```diff
@@ -19,7 +19,6 @@ details-v1:
     - A8_SERVICE=details:v1
     - A8_REGISTER=true
     - A8_ENDPOINT_PORT=9080
-    - A8_HEALTHCHECKS=http://localhost:9080/health
   external_links:
```

Comment:
Good catch. This change can be removed. They we removed as a work around for https://github.com/amalgam8/amalgam8/issues/401, which is now resolved.